lib/ansible/cli/galaxy.py

```diff
@@ -1006,20 +1006,28 @@ def execute_init(self):
             description='your {0} description'.format(galaxy_type),
             ansible_plugin_list_dir=get_versioned_doclink('plugins/plugins.html'),
         )
-        if galaxy_type == 'role':
-            inject_data.update(dict(
-                author='your name',
-                company='your company (optional)',
-                license='license (GPL-2.0-or-later, MIT, etc)',
-                role_name=obj_name,
-                role_type=context.CLIARGS['role_type'],
-                issue_tracker_url='http://example.com/issue/tracker',
-                repository_url='http://example.com/repository',
-                documentation_url='http://docs.example.com',
-                homepage_url='http://example.com',
-                min_ansible_version=ansible_version[:3],  # x.y
-                dependencies=[],
-            ))
+
+        if galaxy_type == "role":
+            if len(str(ansible_version)) <= 3:
```

Comment:
This should use the whole major.minor, rather than checking if ansible_version is 3 chars (which still wouldn't be the case for 2.9.0).
```python 
from ansible.utils.version import SemanticVersion, LooseVersion

if galaxy_type == 'role':
     semantic_version = SemanticVersion.from_loose_version(LooseVersion(ansible_version))
     min_ansible_version=f"{semantic_version.major}.{semantic_version.minor}"
     inject_data.update(...)
```

This also needs a [changelog](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs).

Reply:
Will fix.